torch/nn/quantized/modules/linear.py

```diff
@@ -197,8 +190,10 @@ def __setstate__(self, state):
     def weight(self):
         return torch.ops.quantized.linear_unpack(self._packed_weight)
 
-    def set_weight(self, w):
-        self._packed_weight = torch.ops.quantized.linear_prepack(w)
+    def set_weight_bias(self, w, b):
+        # type: (torch.Tensor, Optional[torch.Tensor]) -> None
+        self._packed_weight = torch.ops.quantized.linear_prepack(w, b)
```

Comment:
Can we call this _packed_weight_and_bias so that the meaning is clear?

Reply:
This is referring more to the `PackedLinearWeight` struct in FBGEMM rather than the actual weight. Also we aren't actually packing the bias with the weight, just adding to the struct. I think changing it to `_packed_weight_and_bias` may cause some misunderstanding.

Reply:
IMO not indicating the presence of the bias at all is worse than someone misunderstanding whether bias is packed or not. Maybe named this something like `_opaque_params`?

Reply:
@jamesr66a @raghuramank100 I'm renaming this to `_packed_params` instead.